src/common/swagger/schema/analytic.yml

```
@@ -31,7 +31,10 @@ components:
           type: number
           example: 50
         change:
-          enum: ['up', 'down']
+          type: string
+          enum: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)